modules/get-started/pages/licensing/monitor-license-status.adoc

```diff
@@ -147,6 +147,40 @@ Redpanda Console displays warnings in the following scenarios:
 
 - *Redpanda Self-Managed enterprise features are in use without a valid license*: If the Redpanda cluster is using enterprise features without a valid license, Redpanda Console displays a warning specifying the features in use without proper licensing.
 
+== Check the license status on the Redpanda resource in Kubernetes
+
+If you have deployed a Redpanda cluster in Kubernetes using the Redpanda Operator, you can monitor the Enterprise Edition license status from the Redpanda custom resource. The operator integrates license monitoring into the resource's `status` field, allowing you to query the license state, expiration, and active enterprise features with `kubectl`.
+
+. Identify the Redpanda resource:
++
+[,bash]
+----
+kubectl get redpanda
+----
++
+Example output:
++
+[.no-copy]
+----
+NAME                  AGE
+example-redpanda      12m
```

Comment:
So, this looks like an old version of the CRDs from what I can tell. The latest, if you install from what's released, should look like this, so no real need to do the JSON path thing below:

```bash
➜  crd-demo git:(main) kubectl get redpanda -A
NAMESPACE   NAME      LICENSE       READY   STATUS
redpanda    cluster   Not Present   True    Redpanda reconciliation succeeded
```

Where the column under `LICENSE` should be, when the cluster is stable, one of `Valid`, `Expired`, or `Not Present`, and, in the unstable case either, `redpanda cluster is not yet ready or up to date` or an error explaining why the license status couldn't be fetched.
```